webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue

```diff
@@ -97,20 +163,24 @@
         </a>
       </div>
 
-      <v-card-text class="spaceCardBody align-center pt-2 pb-1">
+      <v-card-text 
+        class="spaceCardBody align-center"
+        :class="!isMobile && 'pt-2 pb-1'">
```

Comment:
Add variabilized properties first (best practice)

```suggestion
        :class="!isMobile && 'pt-2 pb-1'"
        class="spaceCardBody align-center">
```

Reply:
done!